ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -60,6 +67,25 @@ export class BucketComponent implements OnInit {
     this.fetchBucketTasks();
     this.fetchBucketTasksStates();
     this.fetchBucketTaskPriorities();
+    this.fetchAssigness();
+  }
+
+  showAssigneeList() {
```

Comment:
Nazwa metody nie do konca odzwierciedla to, co ona robi. Lepiej by bylo `toggleShowAssignees`. Unikajmy slow typu list albo items. `showAssignees` znaczy to samo co `showAssigneeListItems`, a jest krotsze, mniejsze cognitive load ;)

Reply:
Dzięki za użycie terminu cognitive load bo podobno kod się pisze dla ludzi :)
Zmieniłem tę nazwę oraz nazwę zmiennej bool do przełączania assigneesów